yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -217,6 +215,20 @@
 </ul>
 </li>
 </ul>
+<t>The "_COMPAT" modifier string is "sticky". Once a revision of a module 
```

Comment:
I think this should be '_COMPAT' if I understand why you're using the single-quotes correctly.  You're trying to match how _COMPAT is referred to above (i.e., in this same section).

Reply:
OK - reverting back. We're a touch inconsistent on the use of double and single quotes in the doc. But it is probably good enough.